frontend/packages/console-shared/src/utils/keyword-filter.ts

```diff
@@ -0,0 +1,19 @@
+export const keywordFilter = <T>(
+  queryString: string,
+  items: T[],
+  compareFunc: (keyword: string, item: T) => boolean,
+): T[] => {
+  if (!queryString) {
+    return items;
+  }
+  const keywords = queryString.trim().split(' ');
+
+  return items.filter((item) => {
+    return keywords.every((keyword) => {
+      if (!keyword) {
```

Comment:
When would `keyword` be empty?

Reply:
I had copied this over from the old function. Doesn't apply anymore, so removed.